source/common/upstream/cluster_manager_impl.cc

```diff
@@ -1523,6 +1523,7 @@ Http::ConnectionPool::InstancePtr ProdClusterManagerFactory::allocateConnPool(
                     {Http::Protocol::Http11, Http::Protocol::Http2, Http::Protocol::Http3}));
 #ifdef ENVOY_ENABLE_QUIC
     Envoy::Http::ConnectivityGrid::ConnectivityOptions coptions{protocols};
+    ASSERT(false);
```

Comment:
Was this intentional? (If yes, add comment?)

Reply:
Nope! Thanks. (Was trying to figure out where this code is called from in production)